app/Models/Chat/Message.php

```diff
@@ -66,29 +66,24 @@ public function getAttribute($key)
         };
     }
 
-    public function reportableAdditionalInfo(?User $reporter): ?string
+    public function reportableAdditionalInfo(): ?string
     {
-        if (
-            !priv_check_user($reporter, 'ChatChannelRead', $this->channel)->can()
-            && !priv_check_user($reporter, 'ChatChannelJoin', $this->channel)->can()
-        ) {
-            return "reporter doesn't actually have access to the reported message";
-        }
```

Comment:
why was this removed?

Reply:
the included backlog now only shows non-pm messages. And if we're to add this check it should probably be during report creation instead anyway and that's kind of unrelated to this PR.